### PR TITLE
Issues/2441

### PR DESCRIPTION
**Pull request by sctnightcore**
base code : https://github.com/OpenKore/openkore/pull/2426
ty @CoffeeParser

need to be test !

### CODE REVIEW

File: src/Network/Send/kRO/RagexeRE_2015_05_13a.pm [hunk 1]
```diff
@@ -85,4 +89,25 @@ sub new {
...
 }
+
+sub rodex_send_mail {
+	my ($self) = @_;
+
+	my $title = stringToBytes($rodexWrite->{title});
+	my $body = stringToBytes($rodexWrite->{body});
+	my $pack = $self->reconstruct({
+		switch => 'rodex_send_mail',
+		receiver => $rodexWrite->{target}{name},
```
Comment (sctnightcore) [inline src/Network/Send/kRO/RagexeRE_2015_05_13a.pm]: $rodexWrite->{name} ?

Comment (sctnightcore) [reply src/Network/Send/kRO/RagexeRE_2015_05_13a.pm]: fixed : 85c1356

**Comment by Nipodemos** [comment]
the only problem is to test
what server use that ragexeRE you edited?

**Comment by sctnightcore** [comment]
I have tested my server 
@nipo

**Comment by Nipodemos** [comment]
fair enough

**Pull request merged**